lib/splash_screen.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:flutter/material.dart';
```

Comment:
move this file in screen folder, no need for main function and material App start directly with Scaffold.